test/docker_e2e.sh

```diff
@@ -62,32 +62,32 @@ docker rm -f dss-crdb-for-debugging &> /dev/null || echo "No CRDB to clean up"
 
 echo "Starting cockroachdb with admin port on :8080"
 docker run -d --rm --name dss-crdb-for-debugging \
-	-p 26257:26257 \
-	-p 8080:8080 \
-	cockroachdb/cockroach:v21.2.7 start-single-node \
-	--insecure > /dev/null
+-p 26257:26257 \
```

Comment:
Let's not remove the indents on these multi-line commands

Reply:
Got tempered by my ide. Fixed it now.